apps/auto-internal-name/src/locations/ConfigScreen.tsx

```diff
@@ -170,72 +88,21 @@ const ConfigScreen = () => {
           </Paragraph>
         </Box>
 
-        <Box>
-          <Heading as="h3">Configure</Heading>
-          <FormControl id="separator">
-            <FormControl.Label marginBottom="spacingS">Separator</FormControl.Label>
-            <TextInput
-              value={parameters.separator}
-              onChange={(e) => setParameters((prev) => ({ ...prev, separator: e.target.value }))}
-            />
-            <FormControl.HelpText marginTop="spacingS">
-              The separator can be any character or symbol and will append to the entry name.
-            </FormControl.HelpText>
-          </FormControl>
-          <FormControl id="sourceFieldId" isInvalid={isSourceFieldMissing}>
-            <FormControl.Label marginBottom="spacingS" isRequired>
-              Source field ID
-            </FormControl.Label>
-            <Autocomplete
-              selectedItem={parameters.sourceFieldId}
-              items={filteredSourceFields.map((field) => field.name)}
-              onInputValueChange={handleSourceFieldInputChange}
-              onSelectItem={handleSourceFieldIdSelection}
-              placeholder="Search field name"
-            />
-            {isSourceFieldMissing && (
-              <FormControl.ValidationMessage>
-                Source field ID is required
-              </FormControl.ValidationMessage>
-            )}
-            <FormControl.HelpText marginTop="spacingS">
-              The source field ID should be the name of the field that you want to use from the
-              parent entry. This will be applied to any content types that include the same field
-              ID.
-            </FormControl.HelpText>
-          </FormControl>
-        </Box>
-
-        <Flex flexDirection="column" fullWidth>
-          <Heading as="h3">Overrides</Heading>
-          <Paragraph>
-            If an override is needed per content type, select the content type and the field name
-            you wish to use for each entry.
-          </Paragraph>
-          {parameters.overrides?.map((override) => (
-            <OverrideRow
-              key={override.id}
-              contentTypes={contentTypes}
-              overrideItem={override}
-              overrideIsInvalid={overridesAreInvalid[override.id]}
-              overrides={parameters.overrides}
-              setOverrides={setOverrides}></OverrideRow>
-          ))}
-          <Box marginBottom="spacingXl">
-            <Tooltip
-              placement="right"
-              id="tooltip-1"
-              content={maxOverridesReached() ? 'No more content types available.' : undefined}>
-              <Button
-                aria-label="Add override"
-                startIcon={<PlusIcon />}
-                isDisabled={maxOverridesReached()}
-                onClick={() => addOverride()}>
-                Add override
-              </Button>
-            </Tooltip>
-          </Box>
-        </Flex>
+        <ConfigureSection
```

Comment:
💭 I don't see the benefit of moving all of this part to a new component and it's a bit less expressive. "ConfigureSection" is a bit too generic and doesn't tell me what does the component do. I would either create 2 different components to handle the separator and the autocomplete part or leave it as it was.